### PR TITLE
Update README and add useful links

### LINKED ISSUE #3: Update the docs
It might be useful to copy over the z-index project brief and helpful links from the emails to the README.

### PR DESCRIPTION
**Pull request by jreinlein**
Fixes #3.
Updated main README and added directory for additional documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
-# z-index-devtool
-A UCOSP project for building a Z-Index Devtool.
+# Context
```
Comment (gregtatum) [inline README.md]: Convention here is that the first header is the name of the project.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,9 @@
...
+This project is a part of [UCOSP](http://ucosp.ca/) and is being steered by Greg Tatum ([@gregtatum](https://github.com/gregtatum/)).
+
+It is intended to be a _prototype_. Additional documentation will be added to the `docs/` directory.
+
+# Project Description
```
Comment (gregtatum) [inline README.md]: nit: Sub-headers should be h2s, so this should probably be `## Project Description`

**Review by gregtatum — COMMENTED** [review]
Looks good to me except a few nits, but let's have another team member finish the review.

**Comment by jreinlein** [comment]
@bkindersley - I implemented @gregtatum 's review, let me know what you think!

**Comment by bkindersley** [comment]
looks good

**Pull request merged**